husky_components/search/main.js

```diff
@@ -22,8 +21,8 @@ define([], function() {
 
     var templates = {
             skeleton: [
-                '<a class="fa-search fa-flip-horizontal search-icon" href="#"></a>',
-                '<a class="fa-times-circle remove-icon" href="#"></a>',
+                '<span class="fa-search fa-flip-horizontal search-icon"></span>',
+                '<span class="fa-times-circle remove-icon"></span>',
```

Comment:
Would you really make that spans? `<a>` tags would indicate that this is something clickable (making the application more accessible).